crates/noirc_evaluator/src/ssa_refactor/acir_gen/acir_ir/acir_variable.rs

```diff
@@ -727,8 +730,21 @@ impl AcirContext {
         id
     }
 
-    pub(crate) fn brillig(&mut self, _code: Vec<BrilligOpcode>) {
-        todo!();
+    pub(crate) fn brillig(
+        &mut self,
+        code: Vec<BrilligOpcode>,
+        inputs: Vec<AcirVar>,
+        output_len: usize,
+    ) -> Vec<AcirVar> {
+        let b_inputs =
+            vecmap(inputs, |i| BrilligInputs::Single(self.data[&i].to_expression().into_owned()));
```

Comment:
Can we remove the `b_` prefix and spell out `brillig_inputs` or give the variable a different name? Same with `b_outputs`.